### PR TITLE
RT-1.5: Updating neighbor configuration for missing routing-policy and specific afi-safi config

### PR DESCRIPTION
**Pull request by trathod1**
This PR introduces 2 changes:

1. route policy is always added and configured on a per-neighbor basis if RoutePolicyUnderNeighborAfiSafi deviation is not enabled,

2. explicitly enables/disables both address families for each BGP neighbor (ipv4 en. / ipv6 dis. for v4 peers, v4 dis., v6 en. for v6 peers). This change is needed to align with the inheritance model discussed in Remove the default of "enabled" of BGP AFI-SAFI to support BGP inheritance model public#774 -> in the current code both IPv4 and IPv6 are enabled on a protocol level, so in a per-neighbor config we want to disable inheritance of the other protocol family.

Reference- https://github.com/openconfig/featureprofiles/pull/1059

"This code is a Contribution to the OpenConfig Feature Profiles project ("Work") made under the Google Software Grant and Corporate Contributor License Agreement ("CLA") and governed by the Apache License 2.0. No other rights or licenses in or to any of Nokia's intellectual property are granted for any other purpose. This code is provided on an "as is" basis without any warranties of any kind."

### CODE REVIEW

File: feature/bgp/prefixlimit/ate_tests/bgp_prefix_limit_test/bgp_prefix_limit_test.go [hunk 1]
```diff
@@ -130,9 +130,7 @@ func configureDUT(t *testing.T, dut *ondatra.DUTDevice) {
 		fptest.AssignToNetworkInstance(t, dut, p1, *deviations.DefaultNetworkInstance, 0)
 		fptest.AssignToNetworkInstance(t, dut, p2, *deviations.DefaultNetworkInstance, 0)
 	}
-	if *deviations.RoutePolicyUnderNeighborAfiSafi {
-		configureRoutePolicy(t, dut, rplName, rplType)
-	}
+	configureRoutePolicy(t, dut, rplName, rplType)
```
Comment (robshakir) [inline feature/bgp/prefixlimit/ate_tests/bgp_prefix_limit_test/bgp_prefix_limit_test.go:133]: Discussed that this ensures that the route policy itself is created regardless of the state of the deviation.

**Comment by xw-g** [comment]
changes seems good to me. pending internal presubmit tests and then merge if no issues.

**Comment by MarcCharlebois** [comment]
@xw-g any progress on the pre-submit tests?

**Comment by xw-g** [comment]
b/272085856 is the internal presubmit effort.

**Review by xw-g — APPROVED** [review]

**Pull request merged**